src/views/Sale/components/Timer/index.tsx

```diff
@@ -1,6 +1,7 @@
 // External
 import React, { useEffect, useState } from 'react'
 import dayjs from 'dayjs'
+import moment from 'moment-timezone'
```

Comment:
Isn't dayjs UTC plugin sufficient? momentjs is deprecated and has a huge bundle size

Reply:
I've removed a library and have instead mapped the shorthand using my own function as Dayjs has deprecated this.